include/wx/cpp.h

```diff
@@ -140,20 +140,6 @@
 
 #ifdef HAVE_VARIADIC_MACROS
```

Comment:
Note to self: we probably should get rid of this one as well. We still need to have checks for old MSVC versions, but there is almost certainly no need for configure test.